env/test/resources/model1.xml

```diff
@@ -55,7 +55,7 @@
         <attribute name="updatedAt" attributeType="Date" usesScalarValueType="NO" syncable="YES"/>
         <attribute name="uuid" attributeType="String" maxValueString="36" syncable="YES"/>
         <attribute name="zipCode" optional="YES" attributeType="String" maxValueString="100" syncable="YES"/>
-        <relationship name="games" optional="YES" toMany="YES" deletionRule="Nullify" destinationEntity="FIGame" inverseName="location" inverseEntity="FIGame" syncable="YES"/>
+        <relationship name="game" optional="YES" deletionRule="Nullify" destinationEntity="FIGame" inverseName="location" inverseEntity="FIGame" syncable="YES"/>
```

Comment:
Xcode dałby jeszcze `maxCount="1"`, ale my chyba na to nie patrzymy? Tylko na `toMany`?

Reply:
Dopiszę dla pewności.